src/build/dotnet/steps/net-core.steps.yaml

```diff
@@ -0,0 +1,73 @@
+# Steps to build .NET Core Projects
+parameters:
```

Comment:
question: should we parameterise the build configuration?

Reply:
Have added